catalog/admin/admin-poller-service/src/test/java/org/codice/ddf/catalog/admin/poller/AdminPollerTest.java

```diff
@@ -0,0 +1,364 @@
+import ddf.catalog.resource.ResourceNotSupportedException;
+import ddf.catalog.service.ConfiguredService;
+import ddf.catalog.source.CatalogStore;
+import ddf.catalog.source.IngestException;
+import ddf.catalog.source.Source;
+import ddf.catalog.source.SourceMonitor;
+import ddf.catalog.source.SourceUnavailableException;
+import ddf.catalog.source.UnsupportedQueryException;
+import ddf.catalog.source.opensearch.OpenSearchSource;
+
+public class AdminPollerTest {
+
+    public static final String CONFIG_PID = "properPid";
+
+    public static final String EXCEPTION_PID = "throwsAnException";
+
+    public static final String FPID = "OpenSearchSource";
+
+    public static MockedAdminPoller poller;
+
+    @Mock
+    private CatalogFramework catalogFramework;
+
+    @Mock
+    private CatalogStore catalogStore;
+
+    private FilterBuilder filterBuilder;
+
+    private Map<String, CatalogStore> catalogStoreMap;
+
+    private ArrayList<String> publishedPlaces;
+
+    private ArrayList<String> destinations;
+
+    private List<Result> results;
+
+    @Before
+    public void setup() {
+        catalogFramework = mock(CatalogFramework.class);
+        catalogStoreMap = new HashMap<>();
+        catalogStoreMap.put("destination1", new MockCatalogStore("destination1", true));
+        catalogStoreMap.put("destination2", new MockCatalogStore("destination2", true));
+        catalogStoreMap.put("destination3", new MockCatalogStore("destination3", true));
+
+        publishedPlaces = new ArrayList<>();
+        publishedPlaces.add("destination1");
+        publishedPlaces.add("destination3");
+
+        destinations = new ArrayList<>();
+        destinations.add("destination1");
+        destinations.add("destination2");
+
+        Metacard metacard1 = new MetacardImpl();
+        metacard1.setAttribute(new AttributeImpl(RegistryObjectMetacardType.PUBLISHED_LOCATIONS,
+                publishedPlaces));
+        metacard1.setAttribute(new AttributeImpl(RegistryObjectMetacardType.REGISTRY_ID,
+                "registry1"));
+        results = new ArrayList<>();
+        results.add(new ResultImpl(metacard1));
+
+        filterBuilder = new GeotoolsFilterBuilder();
+
+        poller = new AdminPollerTest().new MockedAdminPoller(null,
+                catalogFramework,
+                filterBuilder,
+                catalogStoreMap);
+    }
+
+    @Test
+    public void testAllSourceInfo() {
+        List<Map<String, Object>> sources = poller.allSourceInfo();
+        assertNotNull(sources);
+        assertEquals(2, sources.size());
+
+        assertFalse(sources.get(0)
+                .containsKey("configurations"));
+        assertTrue(sources.get(1)
+                .containsKey("configurations"));
+    }
+
+    @Test
+    public void testSourceStatus() {
+        assertTrue(poller.sourceStatus(CONFIG_PID));
+        assertFalse(poller.sourceStatus(EXCEPTION_PID));
+        assertFalse(poller.sourceStatus("FAKE SOURCE"));
+    }
+
```

Comment:
Ah shoot. I changed them on the tests that I wrote but didn't change them elsewhere.
Thoughts? Should I change them here too?